compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposePanel.desktop.kt

```diff
@@ -46,8 +47,29 @@ import org.jetbrains.skiko.SkiaLayerAnalytics
 class ComposePanel @ExperimentalComposeUiApi constructor(
     private val skiaLayerAnalytics: SkiaLayerAnalytics,
 ) : JLayeredPane() {
+    private var reducePresentationLatency: Boolean? = null
```

Comment:
Agree with Ivan that `vsyncEnabled` is better. `reducePresentationLatency` doesn't hint at any downsides - why would anyone *not* want to reduce presentation latency?

But maybe consider going even farther and give it a (fake) enum, e.g. `PresentationMode.VSYNC`, `PresentationMode.IMMEDIATE` etc, if there's a chance we could have more modes...